.gitignore

```diff
@@ -11,3 +11,5 @@ backend/settings/local.py
 
 webpack-stats.dev.json
 webpack-stats.prod.json
+db.sqlite3
```

Comment:
This shouldn't need to be here, as we don't generate this file anymore. If there's a problem with git tracking the file, then that can be fixed in different ways. Not that big of a deal anyways.